networks/suzuka/suzuka-full-node/src/partial.rs

```diff
@@ -232,23 +239,71 @@ where
 	// ! Currently this only implements opt.
 	/// Runs the executor until crash or shutdown.
 	async fn run_executor(&self) -> Result<(), anyhow::Error> {
+		debug!("SuzukaPartialNode run_executor");
 		// wait for both tasks to finish
 		tokio::try_join!(self.write_transactions_to_da(), self.read_blocks_from_da())?;
 
 		Ok(())
 	}
+
+	/// Runs the maptos rest api service until crash or shutdown.
+	async fn run_maptos_rest(&self) -> Result<(), anyhow::Error> {
+		self.maptos_rest.run_service().await?;
+		Ok(())
+	}
 }
 
 impl SuzukaPartialNode<SuzukaExecutorV1> {
 	pub async fn try_from_env(
 	) -> Result<(Self, impl Future<Output = Result<(), anyhow::Error>> + Send), anyhow::Error> {
 		let (tx, _) = async_channel::unbounded();
 		let light_node_client = LightNodeServiceClient::connect("http://0.0.0.0:30730").await?;
+
 		let executor = SuzukaExecutorV1::try_from_env(tx)
 			.await
 			.context("Failed to get executor from environment")?;
-		// TODO: switch to real settlement client
-		let settlement_client = MockMcrSettlementClient::new();
-		Self::bound(executor, light_node_client, settlement_client)
+
+		let settlement_client = build_settlement_client().await?;
+		let maptos_rest = MaptosRest::try_from_env(Some(executor.executor.context.clone()))?;
+		Self::bound(executor, light_node_client, settlement_client, maptos_rest)
 	}
 }
+
+pub const ETH_RPC_URL_ENV_VAR: &'static str = "ETH_RPC_URL";
+pub const ETH_WS_URL_ENV_VAR: &'static str = "ETH_WS_URL";
+pub const ETH_MCR_CONTRACT_ADDRESS_VAR: &'static str = "ETH_MCR_CONTRACT_ADDRESS";
+pub const ETH_DEFAULT_TX_GAS_LIMIT_VAR: &'static str = "ETH_DEFAULT_TX_GAS_LIMIT";
+pub const ETH_MAX_TX_SEND_RETRY_VAR: &'static str = "ETH_MAX_TX_SEND_RETRY";
+pub const ETH_VALIDATOR_PRIVATE_ADDRESS_VAR: &'static str = "ETH_VALIDATOR_PRIVATE_ADDRESS";
```

Comment:
To avoid repetitive and inconsistent `_ENV_VAR` and `_VAR` suffixes in Rust names, it might be better to group these constants under an `env_vars` module, like done in #116.

In general, Rust's module system and name aliases help avoid the C-like "every unit's name must contain its entire provenance and purpose" disease.